userService.js

```diff
@@ -0,0 +1,130 @@
+const mssql = require("mssql/msnodesqlv8");
+const dbConfig = require("./dbconfig");
+const sqlHelper = require("./userServiceSQLHelper");
+
+const pool = new mssql.ConnectionPool(dbConfig.config);
+
+function paginateUsers(req, res) {
+    pool.connect(() => {
+        var sql = "SELECT * FROM dbo.Users";
+        pool.query(sql, function(err, result) {
+            var page = req;
```

Comment:
pagination should be done using SQL
https://www.postgresql.org/docs/8.3/queries-limit.html (same for any sql  DB)

Reply:
Could you please give me a tiny hint about "how"?

Reply:
Removed this method altogether, since was unused.
For pagination implementation please see the `getUsers` method and the Offset/Fetch variables.